.gitignore

```diff
@@ -5,6 +5,7 @@ config/
 
 # sbt specific
 .sbtopts
+.bsp/
```

Comment:
sbt 1.4.x now supports BSP: https://contributors.scala-lang.org/t/build-server-protocol-in-sbt/4234/27 so we need to ignore this generated directory.